### DISCUSSION TITLE
Visual Studio Code Extension

### DISCUSSION THREAD

**Discussion by drummachina** [200ok-ch/organice#973] [Ideas]
Hi, I would like to use Organice to handle my work tasks. I don't want to expose my work tasks to external cloud storage. At the same time, I want to have a decent backup for the tasks. 
I tried to run a local WebDav server, but it couldn't connect from the browser. Maybe because it wasn't accessible from my workstation. Don't know.
I wonder if you consider making an extension for Visual Studio Code? 
There are some extensions for Org Mode, but it is quite limited.

**Comment by schoettl** [comment]
Hi @drummachina,

Have you looked into the [FAQ on WebDAV](https://organice.200ok.ch/documentation.html#faq_webdav)? I haven't tried it myself though.

I don't think that any of the maintainers is motivated to work on a very specific VSCode extension.
But anyone who want to give it a try can jump in.

Otherwise, this issue will be closed as per https://github.com/200ok-ch/organice/blob/master/CONTRIBUTING.org#how-we-work-with-issues
We try to keep the number of issues manageable.

**Comment by drummachina** [comment]
Hi @schoettl,
Thank you for the reply. 
I will try to perform WebDAV troubleshooting you linked. Maybe I missed something.
